patches/minecraft/net/minecraft/entity/EntityList.java.patch

```diff
@@ -63,7 +63,24 @@
      @Nullable
      public static Entity func_191304_a(@Nullable Class <? extends Entity > p_191304_0_, World p_191304_1_)
      {
-@@ -168,13 +180,15 @@
+@@ -150,8 +162,16 @@
+         {
+             return null;
+         }
++        net.minecraftforge.fml.common.registry.EntityEntry entry = net.minecraftforge.fml.common.registry.EntityRegistry.getEntry(p_191304_0_);
++        if (entry != null)
++        {
++            return entry.newInstance(p_191304_1_);
++        }
+         else
+         {
++            net.minecraftforge.fml.common.FMLLog.warning(
```

Comment:
Based on #3953, use `net.minecraftforge.fml.common.FMLLog.log.warning()` and use `{}` instead of `%s`.